PrerequisitesCheckerGramplet/PrerequisitesCheckerGramplet.py

```diff
@@ -0,0 +1,1623 @@
+        #if current version > installed version then don't mention current version running test or development version?
+
+        https://gramps-project.org/wiki/index.php?title=Previous_releases_of_Gramps
+        '''
+        self.append_text("\n")
+        # Start check
+        LATEST_GRAMPS_VERSION = (4, 2, 8)
+        LATEST_GRAMPS_DATE    = "2018-02-09"
+        latest_release_message = "Gramps " + str(LATEST_GRAMPS_VERSION[0]) + "." + str(LATEST_GRAMPS_VERSION[1]) + "." + str(LATEST_GRAMPS_VERSION[2]) + "  released " + LATEST_GRAMPS_DATE + " is the most current version.\n"
+
+        try:
+            try:
+                from gramps.gen.const import VERSION, VERSION_TUPLE
+                gramps_str = VERSION
+                gramps_ver = VERSION_TUPLE
+            except:
+                # Gramps 3.x series
+                from const import VERSION, VERSION_TUPLE
+                gramps_str = VERSION
+                gramps_ver = VERSION_TUPLE
+        except:
+            gramps_str = 'not found'
+
+        # Test
+        if not gramps_ver >= LATEST_GRAMPS_VERSION:
+            #print("Failed")
+            result = """You have Gramps %s . Please make a backup and then upgrade.\n%s""" % (gramps_str, latest_release_message)
+        else:
+            if not gramps_ver == LATEST_GRAMPS_VERSION:
+                result = """You have Gramps %s installed; an unreleased development version. Thank you for contributing and testing; please report any issues. Backups are your friend.\n""" % (gramps_str)
+                #print(gramps_ver)
+            else:
+                #print("Success")
+                #TODO add further check that this is true?
+                result = """You have Gramps %s . Congratulations you have the current version.\n""" % (gramps_str)
+
+        # End check
+        self.render_text(result)
+        #self.append_text(result)
+
+    #Requirements
+    def check1_python(self):
+        '''Check python version Gramps is currently running with against min version required
+
+        #TODO - does not handle an older version of Gramps using python 2 on that a system that has python 3 installed (same for each of the other test
+        '''
+        # Start check
+        MIN_PYTHON_VERSION = (3, 2, 0)
+        min_py_str = str(MIN_PYTHON_VERSION[0]) + "." + str(MIN_PYTHON_VERSION[1]) + "." + str(MIN_PYTHON_VERSION[2])
+
+        # version to check against
+        # Gramps running version of python
+        py_str = '%d.%d.%d' % sys.version_info[:3]
+        check1= "* Python "
+
+        if not sys.version_info >= MIN_PYTHON_VERSION:
+            #print("Failed")
+            messagefailed1 = " (Requires version "
+            messagefailed3= " or greater installed.)\n"
+
+            messagefailed= messagefailed1 + min_py_str + messagefailed3
+
+            result= check1 + py_str + messagefailed
+        else:
+            #print("Success")
+            messagesuccess1 = " (Successs version "

```

Comment:
Successs  > Success

Reply:
Thanks fixed in https://github.com/gramps-project/addons-source/pull/144